andela/AndelaEatsUI
# feat(edit-menu): implement menu editing

#### What does this PR do?
- Enables an admin to edit a menu
#### Description of Task to be completed?
- [x] Add reducer, actions creators and thunk to handle editing a menu
- [x] Modify handleSubmit method to handle both editing and creating a menu
- [x] Ensure menu form is filled with menu data when editing.
#### How should this be manually tested?
- Pull this branch `git fetch origin ft-edit-menu-163282717`
- Login into the application
- Navigate to the admin dashboard and click on menus
- Select a menu to edit
- You should see a modal with a form filled with the menu data
- Edit any preferred field and click `Update menu`
#### What are the relevant pivotal tracker stories?
- [#163282717](https://www.pivotaltracker.com/story/show/163282717)

#### Background Context
N/A
#### Screenshots (If Applicable)
<img width="667" alt="screenshot 2019-01-23 at 08 27 14" src="https://user-images.githubusercontent.com/39955261/51585004-c3ef5a00-1ee8-11e9-96ad-4c85a3aad9bb.png">
